09_reasoning_about_code/build.sbt

```diff
@@ -0,0 +1,10 @@
+scalaVersion := "3.1.1"
+
+val root =
+  crossProject(JSPlatform, JVMPlatform)
+    .jsSettings(
+      libraryDependencies ++=
+        Seq(
+          ("org.scala-js" %%% "scalajs-dom" % "1.1.0").cross(CrossVersion.for3Use2_13),
+        )
+    )
```

Comment:
Previously, there was a file `index.html` to see the program output, it seems to be missing now?